app/parsers/event_parser.rb

```diff
@@ -0,0 +1,53 @@
+class EventParser
+  MissingMeetupDate = Class.new(StandardError)
+
+  def initialize(data)
+    @data = ActiveSupport::HashWithIndifferentAccess.new(data)
+  end
+
+  def parse
+    Event.new(
+      address: venue[:address_1],
+      city: venue[:city],
+      ends_at: end_time_utc,
+      group_name: data[:group][:urlname],
```

Comment:
Is it possible that `data` might not include a `group` key?

Have you considered using [`Hash#dig`](https://docs.ruby-lang.org/en/2.4.0/Hash.html#method-i-dig)?

Reply:
My expectation is the event response will always include information about group hosting the event in the `group` key. The MeetupAPI has proven to be inconsistent but not in this area (at least, not yet), so I'm hesitant to handle such an edge case this eary on.

If the MeetupAPI does change or breaks this expectation, it would be nice to see a helpful error message. The current approach raises the less than helpful type error: `no implicit conversion of Symbol into Integer`. I'll update this to use `fetch` to so the error message reads `key not found :group`.